codespell_lib/data/dictionary.txt

```diff
@@ -892,8 +892,11 @@ againt->against
 againts->against
 agaisnt->against
 agaist->against
+agancies->agencies
+agancy->agency
 aganda->agenda
```

Comment:
I wonder if `aganda` should also suggest `Uganda` ? :shrug:

Reply:
It's another pretty big stretch, and not one of mine, but added anyway.